addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -737,6 +737,25 @@ def test_mailing_shortener(self):
                     link_params=link_params,
                 )
 
+    @users('user_marketing')
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_process_mailing_queue_without_html_body(self):
```

Comment:
There is a class ``TestMassMailValues`` already checking bits about body manipulation, could go in that class :)